graphics/fontforge/Portfile

```diff
@@ -65,39 +67,34 @@ platform darwin 8 {
     configure.env-append PATH=${prefix}/libexec/gnubin/:$env(PATH)
 }
 
-configure.args      --mandir=${prefix}/share/man \
-                    --disable-python-scripting \
-                    --disable-python-extension \
-                    --without-x \
-                    --enable-woff2 \
-                    --disable-silent-rules
+configure.args-append -DENABLE_PYTHON_SCRIPTING=False \
+                      -DENABLE_PYTHON_EXTENSION=False \
+                      -DENABLE_GUI=False \
+                      -DENABLE_WOFF2=True
 
 variant python37 conflicts python38 description {Enable Python support (Python 3.7)} {
     depends_lib-append      port:python37
-    configure.args-delete   --disable-python-scripting \
-                            --disable-python-extension
-    configure.python        ${prefix}/bin/python3.7
-    configure.args-append   am_cv_python_pythondir=${frameworks_dir}/Python.framework/Versions/3.7/lib/python3.7/site-packages \
-                            am_cv_python_pyexecdir=${frameworks_dir}/Python.framework/Versions/3.7/lib/python3.7/site-packages
+    configure.args-delete   -DENABLE_PYTHON_SCRIPTING=False \
+                            -DENABLE_PYTHON_EXTENSION=False
```

Comment:
You should probably use `configure.args-replace` and set the value to `True`, else it may not always be reproducible.

Reply:
I didn’t do this initially because FontForge’s own build defaults to these being on, but at your suggestion, I made this explicit, and made the other options that the CMake build makes configurable but that MacPorts doesn’t (and doesn’t need to) provide knobs for explicit too.